srv/salt/ceph/salt/crc/default.sls

```diff
@@ -1,4 +1,4 @@
-/etc/salt/minion:
+/etc/salt/master.d/crc_method:
   file.append:
```

Comment:
Whoops...that should've been part of the review:
`file.append` seems like the wrong method here. Conceptually this file is fully under our control so `file.managed` with a `content:` section would make more sense. Not sure what file.append does if someone appends a line and re-runs the salt state...append the line again? Certainly a bit hair-split-y...

Reply:
good point, I'll adapt it.